ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/AstProcessorApplication.java

```diff
@@ -0,0 +1,23 @@
+package edu.usb.argos.ASTProcessor;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.List;
+
+@SpringBootApplication
+public class AstProcessorApplication {
+
+	public static void main(String[] args) {
```

Comment:
for this class please remove any print logs, the main method should look like the following:
public static void main(String[] args) {
        SpringApplication.run(AstProcessorApplication.class, args);
    }

Reply:
It was corrected